act-rules-format.bs

```diff
@@ -122,13 +122,13 @@ Language, either written or spoken, as available in nodes of the DOM or accessib
 The means by which the language is assessed, be it by a person or a machine, is not of importance as long as the assessment meets the criteria defined in [[wcag2-tech-req#humantestable]].
 
 
-ACT Test Procedure {#test-proc}
+ACT Test Definition {#test-def}
 ===============================
 
 Applicability
 -------------
 
-The applicability section is a required part of an ACT rule. It MUST contain a precise description of the parts of the test subject to which the rule applies. For example, specific nodes in the DOM tree, or tags that are incorrectly closed in an HTML document. These are known as the *test targets*.
+The applicability section is a required part of an ACT rule. It MUST contain a precise description of the parts of the test subject to which the rule applies. For example, specific nodes in the DOM tree, or tags that are incorrectly closed in an HTML document. These are known as the *test targets*. The applicability MUST only use information provided through test aspects. No other information should be used in the applicability.
```

Comment:
Should say "through test aspects of the same rule" and should link to test aspects section.